src/grammar.coffee

```diff
@@ -154,7 +154,11 @@ URL_VALIDATOR = /// ^
             )
         | git/ (
               refs
-            | refs / heads (/[^/]+)?
+            | refs / (
+                .+
+                | heads (/[^/]+)?
```

Comment:
I guess I maybe don't need heads or tags anyway?

The api allows for instance for this anyway https://api.github.com/repos/philschatz/octokat.js/git/refs/h